include/GafferScene/Instancer.h

```diff
@@ -48,6 +50,52 @@ class GAFFERSCENE_API Instancer : public BranchCreator
 
 	public :
 
+
+		/// Compound plug for representing an image format in a way
+		/// easily edited by users, with individual child plugs for
+		/// each aspect of the format.
+		class ContextVariablePlug : public Gaffer::ValuePlug
+		{
+
+			public :
+
+				GAFFER_PLUG_DECLARE_TYPE( ContextVariablePlug, InstancerContextVariablePlugTypeId, Gaffer::ValuePlug );
+
+				ContextVariablePlug(
+					const std::string &name = defaultName<ContextVariablePlug>(),
+					Direction direction=In,
+					unsigned flags = Default,
+					bool defaultEnable = true
+				);
```

Comment:
Hmmm, I don't see any need for a nameDefault to be specified - the source prim vars are all completely user determined, so the reasonable default is always "".  So if I follow your recommended order, it would be `ContextVariablePlug( enabledDefault, plugName, direction, flags )`.  Putting the optional defaultEnable flag first feels weird to me, but I don't mind if you prefer this order.

Reply:
I was thinking of usage from scripts, where adding a ContextVariablePlug would be quicker if you specified the name etc in the constructor. Maybe that's not as important as it is for NameValuePlug. Maybe `ContextVariablePlug( plugName, direction, enabledDefault, flags )` then, as that's the order for "standard" constructors?

Reply:
Addressed.